packages/web/src/js/Tooltip.ts

```diff
@@ -99,6 +146,16 @@ class Tooltip extends BaseComponent {
     );
   }
 
+  getTipTooltipWrapper() {
+    const id = this.tip.getAttribute('id');
+
+    const triggerWrapperElement = document
+      .getElementById(id!)
```

Comment:
```suggestion
      .getElementById(id)
```

Please, do not suppress the type warnings. Rather use type cast (`as string`) or create a condition to check the value. :-)

Reply:
`!` removed